packages/core/admin/admin/src/pages/MarketplacePage/tests/index.test.js

```diff
@@ -262,9 +262,28 @@ describe('Marketplace page', () => {
     expect(notInstalledText).toBeVisible();
   });
 
-  it('Its showing the sort by menu and its options', async () => {
+  it('Its showing the sort by menu', async () => {
     render(App);
     const sortMenu = screen.getByText('Sort by');
     expect(sortMenu).toBeVisible();
   });
+
+  it('Sort select shows the correct options', async () => {
+    const { container } = render(App);
+    fireEvent.mouseDown(container.querySelector('#marketplaceSortButton'));
```

Comment:
I'm curious to know why you choose to use querySelector here and not a jest selector